_data/simple-icons.json

```
@@ -2430,6 +2430,11 @@
             "hex": "525DDC",
             "source": "https://roots.io/"
         },
+        {
+            "title": "Royal Dutch Shell",
+            "hex": "FFD500",
+            "source": "https://de.wikipedia.org/wiki/Datei:Royal_Dutch_Shell.svg"
```

In this case I think it is more appropriate to use the English wikipedia 😉 

```suggestion
            "source": "https://en.wikipedia.org/wiki/File:Shell_logo.svg"
```